forge/routes/api/device.js

```diff
@@ -653,5 +693,13 @@ module.exports = async function (app) {
         device.targetSnapshotId = deviceSettings?.targetSnapshot
         return true
     }
+    async function assignDeviceToApplication (device, application) {
+        await device.setApplication(application)
+        // Set the target snapshot to match the project's one
+        // TODO
```

Comment:
I assume this means 'todo later once we figure out how to handle application settings (if at all)' rather than needed for 1st iteration?